modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -1455,6 +1496,7 @@ else if (joiningNodeBlt.id() < clusterBlt.id()) {
         U.log(
             log,
             "Sending " + prettyString(state) +
+                ", forceDeactivation=" + forceDeactivation +
```

Comment:
It seems to me, this line should be added after ` request from node [id=" + ctx.localNodeId() +`.
`Sending activation request from node[id=..., topVer=..., froceDeactivation=..., client=...]`